regparser/web/jobs/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+from regparser.web.jobs import views
+from rest_framework.urlpatterns import format_suffix_patterns
+
+urlpatterns = [
+    url(r'^job/$', views.JobViewList.as_view()),
+    url(r'^job/(?P<job_id>[-a-z0-9]+)/$',
```

Comment:
Heh, I had it that way in an earlier version and then simplified it, but am fine with changing it back.